Fix sending error in TestPostProductEvent

Seems the error is due to a bug in gochan.New() in the github.com/cloudevents/sdk-go/v2/protocol/gochan package.

The error manifests itself by the error message: "Sender has not been set." As in, the gochan.New() client has not a functioning Send mechanism?

Hmm manually sending it through as obj interface{} makes obj.(protocol.Sender) fail with:
```
panic: interface conversion: *gochan.SendReceiver is not protocol.Sender: missing method Send [recovered]
	panic: interface conversion: *gochan.SendReceiver is not protocol.Sender: missing method Send
```
While obj.(protocol.Receiver) works as expected (as commented above)

--- a/pkg/mms/events_test.go
+++ b/pkg/mms/events_test.go
@@ -92,18 +92,16 @@ func TestListProductEvents(t *testing.T) {
 	}
 }
 
-// TODO
-// func TestPostProductEvent(t *testing.T) {
-// 	c := newMockCloudeventsClient()
+func TestPostProductEvent(t *testing.T) {
+	c := newMockCloudeventsClient()
 
-// 	event := ProductEvent{}
+	event := ProductEvent{ProductionHub: "test-hub", Product: "test"}
+	err := c.PostProductEvent(&event, Options{})
 
-// 	err := c.PostProductEvent(&event, Options{})
-
-// 	if err != nil {
-// 		t.Errorf("Expected no errors; Got this error: %s", err)
-// 	}
-// }
+	if err != nil {
+		t.Errorf("Expected no errors; Got this error: %s", err)
+	}
+}
 
 // EventClient that sends and receives events on an internal go channel.
 func newMockCloudeventsClient() *EventClient {

